internal/x/format/base_visitor.go

```diff
@@ -138,12 +138,15 @@ func (v *baseVisitor) POptions(isFieldOption bool, options ...*proto.Option) {
 		if len(o.Constant.Array) == 0 && len(o.Constant.OrderedMap) == 0 {
 			v.PWithInlineComment(o.InlineComment, prefix, o.Name, ` = `, o.Constant.SourceRepresentation(), suffix)
 		} else if len(o.Constant.Array) > 0 { // both Array and OrderedMap should not be set simultaneously, need more followup with emicklei/proto
-			// TODO https://github.com/uber/prototool/issues/59
+			v.Failures = append(
+				v.Failures,
+				text.NewFailuref(o.Position, "INVALID_PROTOBUF", "top-level options should never be arrays, this should not compile with protoc"),
+			)
 		} else { // len(o.Constant.OrderedMap) > 0
 			v.P(prefix, o.Name, ` = {`)
 			v.In()
 			for _, namedLiteral := range o.Constant.OrderedMap {
-				v.pInnerLiteral(namedLiteral.Name, *namedLiteral.Literal)
+				v.pInnerLiteral(namedLiteral.Name, *namedLiteral.Literal, "")
```

Comment:
nit: consider `v.pInnerLiteral(namedLiteral.Name, *namedLiteral.Literal, "" /* suffix */)`

Reply:
Is this necessary here? The function definition is right below 10 lines down, and I think this just muddies the water personally :-)

Reply:
Yea, not necessarily. Since the definition is so local it's probably overkill.